cmd/create/cluster/cmd.go

```diff
@@ -941,6 +941,83 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	if billingAccount == "" && !interactive.Enabled() {
+		billingAccount = aws.GetJumpAccount("local")
+		r.Reporter.Infof("Using '%s' as billing account", billingAccount)
+		r.Reporter.Infof("To use a different billing account, add --billing-account xxxxxxxxxx to previous command")
+	}
+
+	if interactive.Enabled() {
+		cloudAccounts, err := r.OCMClient.GetBillingAccounts()
+		if err != nil {
+			r.Reporter.Errorf("%s", err)
+			os.Exit(1)
+		}
+
+		if len(cloudAccounts) > 0 {
+			billingAccount, err = interactive.GetOption(interactive.Input{
+				Question: "Billing Account",
+				Help:     cmd.Flags().Lookup("billing-account").Usage,
+				Default:  billingAccount,
+				Required: true,
+				Options:  cloudAccounts,
+			})
+
+			billingAccount = ocm.RegexpReturnOnlyInteger(billingAccount)
+
+			if err != nil {
+				r.Reporter.Errorf("Expected a valid billing account: '%s'", err)
+				os.Exit(1)
+			}
+		}
+
+		if billingAccount != "" && !ocm.IsValidAWSAccount(billingAccount) {
+			r.Reporter.Errorf("Expected a valid billing account")
+			os.Exit(1)
+		} else {
+			format := "Jan 02, 2006"
```

Comment:
nit: I would move this variable closer to its usage, if possible.

Reply:
Moved.